Source/CesiumRuntime/Private/CesiumGeoreference.cpp

```diff
@@ -573,7 +574,7 @@ void ACesiumGeoreference::_performOriginRebasing() {
     int32 newX = clampedAdd(cameraLocation.X, originLocation.X);
     int32 newY = clampedAdd(cameraLocation.Y, originLocation.Y);
     int32 newZ = clampedAdd(cameraLocation.Z, originLocation.Z);
-    world->SetNewWorldOrigin(FIntVector(newX, newY, newZ));
+    world->RequestNewWorldOrigin(FIntVector(newX, newY, newZ));
```

Comment:
This looks like a subtle change, but it _might_ affect the behavior in a way that I currently cannot foresee. (I know that there was an issue in an internal project where this change solved a bug, but I'd have another close look at this at some point...)

Reply:
Thanks for making a note of that - will be good to test if this change causes any issues. I do know that the Unreal API uses RequestNewWorldOrigin in the `SetWorldOriginLocation` blueprint function.